README.md

```diff
@@ -1,2 +1,57 @@
 # user-activity-monitoring-scripts
 Scripts for the User Activity Monitoring plugin
+
+## User guide
+In order to generate such reports across multiple masters, we have prepared two scripts:
```

Comment:
Missing context of `such reports`. Also, there appear to be three scripts, not two.